Argo/Globals/maybe.swift

```diff
@@ -0,0 +1,6 @@
+public func maybe<A, B>(b: B, f: A -> B, a: A?) -> B {
```

Comment:
`maybe` seems like a weird name here. Should this be called `optional`? Also, the type signature feels un-swifty to me.

Also, why isn't this just `f <^> a ?? b`?